quiver_engine/server.py

```diff
@@ -24,7 +24,16 @@
 
 def get_app(model, temp_folder='./tmp', input_folder='./'):
     get_evaluation_context = get_evaluation_context_getter()
-    single_input_shape = model.get_input_shape_at(0)[1:3]
+
+    if keras.backend.backend() == 'tensorflow':
+        single_input_shape = model.get_input_shape_at(0)[1:3]
+        input_channels = model.get_input_shape_at(0)[3]
+    elif keras.backend.backend() == 'theano':
+        single_input_shape = model.get_input_shape_at(0)[2:4]
+        input_channels = model.get_input_shape_at(0)[1]
+    else:
```

Comment:
This else is unnecessary - afaik .backend() will never return anything else, I'd just change the elif to an else